app/assets/javascripts/assets/wopi/create_wopi_file.js

```diff
@@ -1,3 +1,5 @@
+const FILENAME_MAX_LENGTH = 100;
```

Comment:
Add Constants::FILENAME_MAX_LENGTH to assets/javascripts/sitewide/constants.js.erb and use it here from there

Reply:
Thanks, I added it.